frontend/controller/service-worker.js

```diff
@@ -341,6 +373,9 @@ sbp('sbp/selectors/register', {
 sbp('sbp/selectors/register', {
   'gi.notifications/*': swRpc
 })
+sbp('sbp/selectors/register', {
+  'sw/*': swRpc
+})
```

Comment:
Is there a reason we're calling `'sbp/selectors/register'` 7 times here?

All of these selectors could be put into a single call to `'sbp/selectors/register'`...

Reply:
I thought the call to `sbp/selectors/register` were supposed to be for a single domain?

Reply:
I think it can handle multiple domains, but I suppose yeah.. that is the convention... fair enough

Reply:
> I think it can handle multiple domains

Yeah, it can.